src/kaleidoscope/hardware/kbdfans/KBD4x.h

```diff
@@ -24,9 +24,19 @@
 #include "Kaleidoscope-HIDAdaptor-KeyboardioHID.h"
 
 #include "kaleidoscope/macro_helpers.h"
-#include "kaleidoscope/hardware/avr/pins_and_ports.h"
+
+struct cRGB {
+  uint8_t g;
+  uint8_t r;
+  uint8_t b;
+};
+
+#define CRGB(r, g, b) (cRGB){g, r, b}
 
 #include "kaleidoscope/hardware/ATMegaKeyboard.h"
+#include "kaleidoscope/driver/led/WS2812.h"
+
+using Color = kaleidoscope::driver::led::color::GRB;
```

Comment:
...same as above. But also, now I wonder if we could add helpers for the standard color orders, to reduce the chance of implementation code drift. So that you could just declare the color order and get your CRGB, cRGB, etc. It might also make it easier when we have to do cRGBW...

Reply:
Yup! I have a half-baked branch that does just that. I'd keep that separate, because that's a fairly invasive change.